src/components/LinterMessage/index.spec.tsx

```diff
@@ -141,7 +149,23 @@ describe(__filename, () => {
   it('can be marked as an inline message', () => {
     const root = render({ inline: true });
 
-    expect(root).toHaveClassName(`.${styles.inline}`);
+    expect(root.find(Alert)).toHaveClassName(`.${styles.inline}`);
+  });
+
+  it('calls _scrollToElement() when requested', () => {
+    const _scrollToElement = jest.fn();
+
+    // We need `mount()` because `ref` is only used in a DOM environment.
+    const root = mount(
```

Comment:
Hmm, I don't think it's a great idea to use `mount` here. It will introduce complications if `LinterMessage` becomes a Redux connected component now or in the future.

Here is an example of a way to test it without `mount`:
https://github.com/mozilla/addons-code-manager/blob/c29a7ddd16ec6b43d84ccc580b415ca3988c259a/src/components/FileTreeNode/index.tsx#L30

https://github.com/mozilla/addons-code-manager/blob/c29a7ddd16ec6b43d84ccc580b415ca3988c259a/src/components/FileTreeNode/index.spec.tsx#L599

Testing it without `mount` isn't perfect because you can't cover the actual `ref=` definition part but maybe that's OK. I guess it depends. Anyway, just throwing this out as an alternative.